lib/agent/socket_util.js

```diff
@@ -306,8 +306,13 @@ function validateCert(cert, cb)
       });
     }
 
-    // if we already have an entry in the cache, use it
-    let ocspResponse = getOcspResponseCache().get(cert);
+    let ocspResponse;
+    // check if cache is initialized before getting entry
+    if (getOcspResponseCache().isInitialized())
```

Comment:
Added a check here because if "get(cert)" is called and the cache is not initialized, then an error is thrown due to cache being 'undefined'